omnibus-base-sepolia-andromeda.toml

```diff
@@ -301,6 +326,44 @@ defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
 defaultValue = "<%= imports.erc_4626_to_assets_ratio_oracle.imports.statausdc_token_mock.contracts.Token.address %>"
 description = "We use the mock generated by erc-4626-to-assets-ratio-oracle package in statausdc-usdc.toml"
 
+[provision.cbbtc_mock_collateral]
+source = "mintable-token:1.8"
+target = "synthetix-mock-tokens:1.8@cbbtc"
+options.name = "Fake Coinbase Bitcoin"
+options.symbol = "fcbbtc"
+options.owner = "<%= settings.owner %>"
+options.decimals = "18"
+
+[provision.cbeth_mock_collateral]
+source = "mintable-token:1.8"
+target = "synthetix-mock-tokens:1.8@cbeth"
+options.name = "Fake Coinbase Wrapped Staked ETH"
+options.symbol = "fcbeth"
+options.owner = "<%= settings.owner %>"
+options.decimals = "18"
+
+[provision.wsteth_mock_collateral]
+source = "mintable-token:1.8"
+target = "synthetix-mock-tokens:1.8@wsteth"
+options.name = "Fake Lido Wrapped Staked Eth"
+options.symbol = "fwsteth"
+options.owner = "<%= settings.owner %>"
+options.decimals = "18"
+
+[provision.weth_mock_collateral]
```

Comment:
we dont need WETH because we can get unlimited ETH and it is one `.deposit()` away from WETH. So please use real WETH

Reply:
would you mind if we keep it as fake? because i hate using real scarce tokens on testnets 😅 , but no preference if you have a strong opinion about it.

Reply:
weth should not be fake. All testnets use real weth.

Reply:
i want to maintain this, i feel the pain of acquiring testnet tokens...

Reply:
It probably is down to our context. I do all the testing in anvil. So getting weth is just `.deposit({value: whatever})` (because we have unlimited ETH) and all our tests are using real ETH -> WETH.

At the same time it is one of the "real" tokens that we can make tests as close to mainnet as possible (no magic).

So at least we can guaranteed have one real token in our testnet deployment,

These are my thoughts pro real WETH.

The only negative in using real WETH I could see if we do extensive testing on real sepolia and not forks, but I don't do it anywhere.

Reply:
yeah alright please let's keep this as fake weth 🙏🏼  it helps those who need to use the chain for testing, has no net negative, the supply of these tokens is being constrained.... so hopefully no one is abusing anymore

Reply:
But if you really really feel otherwise about it, happy to make the necessary updates ofc,

Reply:
I realised there are  more things. 1. Frontend cannot rely on weth being weth and do normal wrapping workflow and will have to treat it differently for testnet. 
2. e2e tests will need to differ for testnet and mainnet for weth in workflow again - testnet will have to use fake token mint instead of deposit. And so far weth tests were almost the only ones that are same for both testnet and mainnet

Reply:
alright will revise then, no worries,